src/gui/wizard/owncloudsetupnocredspage.ui

```diff
@@ -221,18 +221,34 @@
             <item>
              <widget class="QLabel" name="urlLabel">
               <property name="sizePolicy">
-               <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
+               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
+              <property name="minimumSize">
+               <size>
+                <width>24</width>
+                <height>24</height>
+               </size>
+              </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
+          <item>
+           <widget class="QLabel" name="label">
+            <property name="text">
+             <string>This is the link to your Nextcloud web interface when you open it in the browser.&lt;br/&gt; It looks like https://cloud.example.com or https://example.com/cloud</string>
```

Comment:
We should use the Application Name here instead of the hard-coded "Nextcloud"

Reply:
Good point, done.